manager/traverse.go

```diff
@@ -263,10 +264,27 @@ func handleFile(templateIndex map[string]*model.Template, fullPath, relativePath
 		if err != nil {
 			return err
 		}
+
```

Comment:
Wait no, wouldn't want it in that function. Something like `parse.TemplateInfo` might be good though.